docs/source/conf.py

```diff
@@ -18,17 +18,65 @@
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
-
 import os
 import sys
+import tempfile
 
-# Following 3 lines avoid the need of importing load_dbenv() for compiling the documentation -> works also without verdi install
+# Add `aiida` to the path so it can be imported without installing it
 sys.path.append(os.path.join(os.path.split(__file__)[0], os.pardir, os.pardir))
-sys.path.append(os.path.join(os.path.split(__file__)[0], os.pardir))
-os.environ['DJANGO_SETTINGS_MODULE'] = 'rtd_settings'
 
 import aiida
 
+
+def load_documentation_profile():
+    """Load a dummy profile just for the purposes of being able to build the documentation.
+
+    The building of the documentation will require importing the `aiida` package and some code will try to access the
+    loaded configuration and profile, which if not done will except. On top of that, Django will raise an exception if
+    the database models are loaded before its settings are loaded. This also is taken care of by loading a Django
+    profile and loading the corresponding backend. Calling this function will perform all these requirements allowing
+    the documentation to be built without having to install and configure AiiDA nor having an actual database present.
+    """
+    from aiida.manage import configuration
+    from aiida.manage.manager import get_manager
+
+    with tempfile.NamedTemporaryFile() as handle:
+        profile_name = 'readthedocs'
+        profile = {
+            'AIIDADB_ENGINE': 'postgresql_psycopg2',
+            'AIIDADB_BACKEND': 'django',
+            'AIIDADB_PORT': 5432,
+            'AIIDADB_HOST': 'localhost',
+            'AIIDADB_NAME': 'aiidadb',
+            'AIIDADB_PASS': 'aiidadb',
+            'AIIDADB_USER': 'aiida',
+            'AIIDADB_REPOSITORY_URI': 'file:///dev/null',
+        }
+        config = {
+            'default_profile': profile_name,
+            'profiles': {
+                profile_name: profile
+            }
+        }
+        configuration.PROFILE = configuration.Profile(profile_name, profile, from_config=True)
+        configuration.CONFIG = configuration.Config(handle.name, config)
+        get_manager()._load_backend(schema_check=False)
+
+
+# Load the dummy profile even if we are running locally, this way the documentation will succeed even if the current
+# default profile of the AiiDA installation does not use a Django backend.
+load_documentation_profile()
+
+
+# If we are not on READTHEDOCS load the Sphinx theme manually
+if not os.environ.get('READTHEDOCS', None):
+    try:
```

Comment:
try/except should not be needed here since we have the package in the docs dependencies

Reply:
Done